css-cascade-5/Overview.bs

```diff
@@ -1035,6 +1035,7 @@ Cascade Sorting Order</h3>
 			For this purpose:
 
 			<ul>
+				<li>Style sheets are ordered as in <a href="https://www.w3.org/TR/cssom-1/#documentorshadowroot-final-css-style-sheets">final CSS style sheets</a>.
```

Comment:
I left the HREF here for now, and I'll land a followup CL converting it to a bikeshed link after this lands, if that's ok with you.

Reply:
```suggestion
				<li>Style sheets are ordered as in <a href="https://drafts.csswg.org/cssom/#documentorshadowroot-final-css-style-sheets">final CSS style sheets</a>.
```

since the TR URL there is stuck in 2016.

Reply:
Oops, better, thanks.